### ISSUE TITLE
Stripping www is optional

### ISSUE THREAD

**Issue by BurntCaramel** [sindresorhus/normalize-url#10]
This module looks great, nice and light weight. It would be nice, in my opinion, if the stripping of the ‘www’ was optional. A `stripWWW`option could be added.

**Comment by sindresorhus** [comment]
Duplicate of https://github.com/sindresorhus/normalize-url/issues/4

Going to close this unless you have any additional reasoning/use-case.

**Comment by BurntCaramel** [comment]
It’s natural to us developers that www will redirect to the root domain, but this isn’t always the case on every website. Also, on some sites only the home page redirects correctly, and not subpages.

It could be confusing to normal people that the www has suddenly disappeared from the URL they entered into a field.

Also, having the link be forced to be a redirect for some sites (e.g. www.facebook.com) because the www has been taken off has some increase in loading time due to the redirect, and I don’t think so but could have some SEO impact.

Basically it would be nice to link to the *canonical* URL, if that is what I have entered it would be nice if the www-stripper could be optionally disabled.
